libraries/NDP/examples/AlexaDemo/AlexaDemo.ino

```diff
@@ -44,9 +44,9 @@ void setup() {
   NDP.onMatch(ledBlueOn);
   NDP.onEvent(ledGreenOn);
   Serial.println("Loading synpackages");
-  NDP.begin("mcu_fw_120_v91.synpkg");
-  NDP.load("dsp_firmware_v91.synpkg");
-  NDP.load("alexa_334_NDP120_B0_v11_v91.synpkg");
+  NDP.begin("mcu_fw_120.synpkg");
+  NDP.load("dsp_firmware.synpkg");
+  NDP.load("alexa_model334_ndp120.synpkg");
```

Comment:
I suggest we add `_105`suffic to the firmware and model files - this way we help make sure users don't try to use the wrong versions of the files.